packages/ember-routing/lib/system/router.js

```diff
@@ -1291,9 +1297,11 @@ function updatePaths(router) {
 
   let path = EmberRouter._routePath(infos);
   let currentRouteName = infos[infos.length - 1].name;
+  let currentURL = router.get('location').getURL();
 
   set(router, 'currentPath', path);
   set(router, 'currentRouteName', currentRouteName);
+  set(router, 'currentURL', currentURL);
```

Comment:
I would think this change is not needed since we are setting `router.currentURL` above in the changes to `_setupRouter`.